exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,16 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+
+var containsWord = "true"   // creates variable true
+var noWord = "false"        //creates variable false
```

Comment:
declaring global variables like this is encouraged, only if there are universal variables that are useful to multiple functions. In this case, true || false are primitive, and can just be put inside function.